yarn.lock

```diff
@@ -64,6 +64,15 @@
     "@aragon/connect-ethereum" "^0.7.0-beta.3"
     "@aragon/connect-thegraph" "^0.7.0-beta.3"
 
+"@aragon/provided-connector@^6.0.7":
```

Comment:
Just checking – Are we definitely updating the existing lock file? the diff just looks a little large so want to make sure!

Reply:
Never mind, use-wallet seems to be bring plenty of friends 😅

Reply:
yes :)